deployment/7_deploy_2.5.js

```diff
@@ -103,6 +107,7 @@ const main = async () => {
   configurator.updateInfrastructureAddresses({
     WalletFactory: WalletFactoryWrapper.address,
     BaseWallet: BaseWalletWrapper.address,
+    Authoriser: AuthoriserWrapper.address,
```

Comment:
We should add that to the schema as well although might leave it for the cleanup we'll do later.